examples/qualcomm/scripts/utils.py

```diff
@@ -187,6 +187,7 @@ def build_executorch_binary(
         quantizer = QnnQuantizer()
         quantizer.add_custom_quant_annotations(custom_annotations)
         quantizer.set_per_channel_linear_quant(per_channel_linear)
+        quantizer.set_per_channel_conv_quant(True)
```

Comment:
Is it always true or configurable?

Reply:
I think we set it always to true as there are couple of models having bad accuracy due to turning off per_channel_conv.